packages/commonwealth/client/scripts/views/components/Comments/CreateComment.tsx

```diff
@@ -94,8 +94,8 @@ export const CreateComment = ({
         const input = await buildCreateCommentInput({
           communityId,
           address: user.activeAccount!.address,
-          threadId: rootThread.id,
-          threadMsgId: rootThread.canvasMsgId!,
+          threadId: rootThread.id ?? null,
```

Comment:
thread id is required for creating comment, so it will likely fail in these cases

https://github.com/hicommonwealth/commonwealth/blob/bea27b27f6e53cf934b84b0e64c8ce0ee7e73844/libs/schemas/src/commands/comment.schemas.ts#L7

Reply:
I think this actually doesn't need the null option, let me fix that.